src/index.html

```diff
@@ -12,7 +12,42 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
```

Comment:
no needs to add a class selector to a body
```suggestion
  <body>
```